web/styles/app_components.css

```diff
@@ -744,6 +744,11 @@ div.overlay {
         width: 1%;
         white-space: nowrap;
     }
+
+    & th,
+    td {
+        padding: 4px 5px;
+    }
```

Comment:
We can probably remove `table-condensed` class from the HTML as we require paddings in all the tables.

Reply:
That seems reasonable to me, yeah.